File: src/Year2021/Day1/PuzzleTest.php
<?php

namespace AdventOfCode\Year2021\Day1;

use AdventOfCode\TestInterface;

class PuzzleTest implements TestInterface
{
    public function testFirstPart(): iterable
    {
        yield '7' => <<<INPUT
199
200
208
210
200
207
240
269
260
263
INPUT;
    }
    public function testSecondPart(): iterable
    {
        yield '5' => <<<INPUT
199
200
208
210
200
207
240
269
260
263
INPUT;
    }
}

File: python/Challenges/2023/Day15/puzzleTest.py
class PuzzleTest:
    def test_first_part():
        yield {'result': 52, 'test_input': """
HASH
"""
               }
        yield {'result': 1320, 'test_input': """
rn=1,cm-,qp=3,cm=2,qp-,pc=4,ot=9,ab=5,pc-,pc=6,ot=7
"""
               }

    def test_second_part():
        yield {'result': 145, 'test_input': """
rn=1,cm-,qp=3,cm=2,qp-,pc=4,ot=9,ab=5,pc-,pc=6,ot=7
        """
               }

